website/src/app/documentation/demos/page-layouts/page-layouts.demo.html

```diff
@@ -76,6 +85,12 @@ <h4>Demo</h4>
           <span class="nav-text">Command bar</span>
         </a>
       </div>
+      <div>
+        <cds-icon shape="display"></cds-icon>&nbsp;
+        <a class="nav-link" routerLink="/full-page-layouts/action-panel" routerLinkActive="active">
+          <span class="nav-text">Action panel</span>
```

Comment:
also state somewhere how the buttons should be order as a guideline:
`<history button> <custom page specific buttons> <action-panel-button>`